using-timescaledb/compression.md

```diff
@@ -0,0 +1,417 @@
+
+TimescaleDB does not support inserts orupdates into a compressed chunk. To insert 
+or update data, we must first decompress the target chunks. But before we can do 
+that, we need to turn off our compression policy, otherwisethat policy will attempt 
+to re-compress the chunks that we are currently working on (not the desired result). 
+To accomplish this we will issue the following command:
+
+``` sql
+select remove_compress_chunks_policy('conditions');
+```
+
+We have now removed the compress chunk policy from the hypertable conditions which 
+will leave us free to decompressing the chunks we need to modify via backfill or 
+update. To decompress the chunk(s) that we will be modifying, for each chunk:
+
+``` sql
+select decompress_chunk( '_timescaledb_internal._hyper_2_2_chunk'); 
+```
+
+>:TIP: You would want to run this command for each chunk that would be impacted 
+by your INSERT or UPDATE statement in backfilling data. Once your needed chunks 
+are decompressed you can proceed with your data backfill operations.
+
+Once your backfill and update operations are complete we can simply re-enable our compression policy job:
+
+``` sql
+select add_compress_chunks_policy('cpu', '60d'::interval);
+```
+This job will run and re-compress any chunks that you may have decompressed during your backfill operation.
+
+---
+
+### Best Practices [](best)
+Considerations around Compression Policy:
+
+When setting our compression policy and the timing around when to compress a 
+chunk you should consider the types of queries that that you are running. Our 
+experience has shown that when data is young (positions 0-2 in our use case) 
+we tend to query the data in a more shallow and wide manner. 
+
+As an example, show me current CPU usage, disc usage, energy consumption, 
+and I/O for server X. In this case the row based format that is native to PostgreSQL 
+will serve us well from a performance perspective. As data begins to age and our 
+queries being to become more analytical in nature (deep and narrow queries) as an 
+example we might want to calculate the average number of logins across all users. 
+In this case, the columnar nature of the query (performing a function on the number 
+of logins) will lend itself to better performance.  
+
+The process of compression as we have implemented it involves converting row based 
+data into more of a columnar format to achieve better overall data consolidation.  
+In the long term this is one part of how you want to think about your compression 
+policy strategy (i.e. at what point do you start to use deep and narrow queries) 
+along with things like frequency of access and disk savings to decide when to start 
+compressing checks.
+
+>:WARNING: The current release of TimescaleDB supports the ability to query data in 
+compressed chunks, however, it does not support inserts, or updates into compressed 
+chunks.  The next several sections will discuss ways to deal with that limitation.  
```

Comment:
> The next several sections...

Should not be in a warning.  Warnings should be standalone.  If you want to refer to other sections, move this into normal text.

Reply:
Or just remove the last sentence

Reply:
done